posthog/api/capture.py

```diff
@@ -66,9 +66,13 @@ def parse_kafka_event_data(
 def log_event(data: Dict, event_name: str, partition_key: Optional[str]):
     logger.debug("logging_event", event_name=event_name, kafka_topic=KAFKA_EVENTS_PLUGIN_INGESTION_TOPIC)
 
+    # To allow for different quality of service on session recordings and other
+    # events, we push to a different topic.
+    kafka_topic = KAFKA_SESSION_RECORDING_EVENTS if event_name == "$snapshot" else KAFKA_EVENTS_PLUGIN_INGESTION_TOPIC
+
```

Comment:
nit: For consistency in the logs, I'd move the `kafka_topic` variable assignment before the `logger.debug` call and use it as the `kafka_topic` argument.

Reply:
```suggestion
    # To allow for different quality of service on session recordings and other
    # events, we push to a different topic.
    kafka_topic = KAFKA_SESSION_RECORDING_EVENTS if event_name == "$snapshot" else KAFKA_EVENTS_PLUGIN_INGESTION_TOPIC

    logger.debug("logging_event", event_name=event_name, kafka_topic=kafka_topic)
```

Reply:
On a similar fashion, line 80 should also use the `kafka_topic` variable instead of `KAFKA_EVENTS_PLUGIN_INGESTION_TOPIC` (and use the logging module instead of `print` but that's a different discussion...).